cloud-event-dispatcher/src/main/resources/cloud-event-dispatcher-service-openapi.yaml

```diff
@@ -0,0 +1,7510 @@
+openapi: 3.0.3
+info:
+  title: CloudSDK Cloud Event Dispatcher API
+  description: APIs that provide services for creating cloud events events.
```

Comment:
APIs that provide services for posting System Events and WLAN Service Metrics into the cloud.